dbcon/execplan/simplefilter.cpp

```diff
@@ -55,6 +55,15 @@ SimpleFilter::SimpleFilter(const string& sql) : Filter(sql)
   parse(sql);
 }
 
+// TODO: only handled simplecolumn operands for now
+SimpleFilter::SimpleFilter(const string& sql, ForTestPurposesWithoutColumnsOIDS) : Filter(sql)
+{
+  parse(sql, ForTestPurposesWithoutColumnsOIDS{});
+}
+
+
```

Comment:
Nit: It seems like CS keeps 2 blank lines between next function definition. Could you remove unnecessary blank lines.

Reply:
Done